DistFiles/localizations/Chorus.en.tmx

```diff
@@ -31,27 +25,21 @@
         <seg>Add new message:</seg>
       </tuv>
     </tu>
-    <tu tuid="AnnotationEditorView.ResolveNote">
+    <tu tuid="AnnotationEditorView.button1">
```

Comment:
I think this is because we are using the Chorus 2.5 branch because the master branch of Chorus already has this problem corrected.